core/roslib/include/ros/package.h

```diff
@@ -108,10 +108,18 @@ ROSLIB_DECL bool getAll(V_string& packages);
 ROSLIB_DECL void getPlugins(const std::string& package, const std::string& attribute, V_string& plugins, bool force_recrawl=false);
 
 /**
+ * \brief Call the "rospack plugins" command, eg. "rospack plugins --attrib=<attribute> <package>".  Returns a pair of vector of strings which
+ * are package names and export values respectively. This allows for multiple values in the one package name.
+ */
+ROSLIB_DECL void getPlugins(const std::string& package, const std::string& attribute, V_string& packages, V_string& plugins, bool force_recrawl=false);
```

Comment:
Yes, I think adding a new public function with a vector of pairs would be cleaner. And this function (with two vectors) can stay internal. Can you update the PR that way? Updating only one is ok - I can cherry-pick the changes after one has been merged.